tests/image/manifest

```diff
@@ -34,7 +34,7 @@
                 "name": "resource/memory",
                 "value": {
                     "request": "25M",
-                    "limit": "120M"
+                    "limit": "25M"
```

Comment:
Can `bin/actool help patch-manifest` change the value of isolators? If not, maybe we need an issue for this. So we will not have to keep the value in sync between the manifest here and `tests/inspect/inspect.go` file.